### PR TITLE
Prettify Error rendering during code questions and grading page

### PR DESCRIPTION
**Pull request by ankitjavalkar**

### CODE REVIEW

File: yaksh/templates/yaksh/grade_user.html [hunk 1]
```diff
@@ -156,22 +156,27 @@
     {% else %}
     <h5>Student answer: </h5>
     {% for answer in answers %}
-    {% if not answer.skipped %}
-    {% if answer.correct %}
+    {% if not answer.0.skipped %}
```
Comment (prabhuramachandran) [inline yaksh/templates/yaksh/grade_user.html]: Why sis this needed?

**Comment by prabhuramachandran** [comment]
LGTM apart from minor comment.

**Comment by ankitjavalkar** [comment]
Fixing another minor error, will update the PR and leave a comment here.

File: yaksh/templates/yaksh/grade_user.html (hunk 1 above)
Comment (ankitjavalkar) [inline yaksh/templates/yaksh/grade_user.html]: It was already there, have removed it as I guess skipped answers should also be available for review

**Comment by ankitjavalkar** [comment]
Done, changes have been made as expected

**Pull request merged**